config/locales/en.yml

```diff
@@ -477,6 +477,8 @@ en:
               not_a_number_when_many: Total money in bank account %{position} must be a number. Enter 0 if this does not apply.
               greater_than_or_equal_to: Total money in bank account must be 0 or more. Enter 0 if this does not apply.
               greater_than_or_equal_to_when_many: Total money in bank account %{position} must be 0 or more. Enter 0 if this does not apply.
+            account_in_dispute:
```

Comment:
please add a TODO to the related spec

Reply:
I will double check but I think the translation error shows up in several specs with the new `def relevant_steps` method as we are now checking validity a lot more often to know which steps are relevant. And the custom method for bank account errors when it can't find inclusion for account in dispute (never a user facing error hence why there was no translation). If the spec can find the error then it happily passes over the form and returns it as invalid. I can double check which specs are affected and try and mark them? But it could be a few of them

Reply:
I did run it and this is the only test that breaks. it might be worth adding a todo so we review it as it sounds like it might be something that can be resolved in the test setup 🤷